### ISSUE TITLE
Failing test in tracks-on-tracks-on-tracks exercise

### ISSUE THREAD

**Issue by BTowersCoding** [exercism/clojure#484]
The exemplar solution does not pass the following test:

``` clojure
(deftest list-add-test
  (is (= '("JavaScript" "Java" "Lisp" "Clojure")
         (-> (tracks-on-tracks-on-tracks/new-list)
              (tracks-on-tracks-on-tracks/add-language "Clojure")
              (tracks-on-tracks-on-tracks/add-language "Lisp")
              (tracks-on-tracks-on-tracks/add-language "Java")
              (tracks-on-tracks-on-tracks/add-language "JavaScript")))))
```

This is due to the argument order being changed by a PR at one point. It passes if the unit test is changed to use the thread-last macro (`->>`) instead of thread-first (`->`).

However, this brings up another, more disturbing question: Why isn't this being caught by CI? Every exemplar solution for all exercises are tested on every change. I'll create another issue for this.

**Comment by BTowersCoding** [comment]
I propose that the test suite be left unchanged, and the exemplar solution made to be the following:

``` clojure
(ns tracks-on-tracks-on-tracks)

(defn new-list []
  '())

(defn add-language
  [lang-list lang]
  (conj lang-list lang))

(defn first-language
  [lang-list]
  (first lang-list))

(defn remove-language
  [lang-list]
  (rest lang-list))

(defn count-languages
  [lang-list]
  (count lang-list))

(defn learning-list []
  (-> (new-list)
      (add-language "Clojure")
      (add-language "Lisp")
      remove-language
      (add-language "Java")
      (add-language "JavaScript")
      count-languages))
```

**Comment by BTowersCoding** [comment]
This will be fixed by #486
